src/platform/platform_linux.c

```diff
@@ -146,7 +146,7 @@ QuicPlatformInitialize(
 #ifdef QUIC_PLATFORM_DISPATCH_TABLE
     QUIC_FRE_ASSERT(PlatDispatch != NULL);
 #else
-    RandomFd = open("/dev/urandom", O_RDONLY);
+    RandomFd = open("/dev/urandom", O_RDONLY|O_CLOEXEC);
```

Comment:
I don't completely understand the intricacies here. Why is `O_CLOEXEC` being added?

Reply:
Not sure why it was added here by Lars, but it's definitely good hygiene for programs that fork. That being said, I don't know what MsQuics expected behavior is in a process that forks ;P

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/android-cloexec-open.html

Can also just disable this check.

Reply:
I don't have the expertise here so I'll have to you defer to whatever you guys think is best. I believe I originally grabbed this code from stackoverflow.

Reply:
There is IMO no downside to this change. @Maximus- ?

Reply:
I worry that this would break macOS when we get to that PR.